doc/source/release/1.11.0-notes.rst

```diff
@@ -105,25 +267,837 @@ Deprecated features
 ******************************
 Backwards incompatible changes
 ******************************
+- The default for the ``legacy`` keyword of `scipy.special.comb` has changed
+  from ``True`` to ``False``, as announced since its introduction.
+
+********************
+Expired Deprecations
+********************
+There is an ongoing effort to follow through on long-standing deprecations.
+The following previously deprecated features are affected:
+
+- The ``n`` keyword has been removed from `scipy.stats.moment`.
+- The ``alpha`` keyword has been removed from `scipy.stats.interval`.
+- The misspelt ``gilbrat`` distribution has been removed (use
+  `scipy.stats.gibrat`).
+- The deprecated spelling of the ``kulsinski`` distance metric has been
+  removed (use `scipy.spatial.distance.kulczynski1`).
+- The ``vertices`` keyword of `scipy.spatial.Delauney.qhull` has been removed
+  (use simplices).
+- The ``residual`` property of `scipy.sparse.csgraph.maximum_flow` has been
+  removed (use ``flow``).
+- The ``extradoc`` keyword of `scipy.stats.rv_continuous`,
+  `scipy.stats.rv_discrete` and `scipy.stats.rv_sample` has been removed.
+- The ``sym_pos`` keyword of `scipy.linalg.solve` has been removed.
+- The `scipy.optimize.minimize` function now raises an error for ``x0`` with
+  ``x0.ndim > 1``.
+- In `scipy.stats.mode`, the default value of ``keepdims`` is now ``False``,
+  and support for non-numeric input has been removed.
+- The function `scipy.signal.lsim` does not support non-uniform time steps
+  anymore.
+
 
 *************
 Other changes
 *************
+- Rewrote the source build docs and restructured the contributor guide.
+- Improved support for cross-compiling with meson build system.
+- MyST-NB notebook infrastructure has been added to our documentation.
 
 
 
 *******
 Authors
 *******
 
+* h-vetinari (69)
```

Comment:
Other single-name users seem to be sorted alphabetically, not sure what's going on there; I'm also _really_ surprised by my supposed commit count, it should be around half that:
```
git shortlog -sn --no-merges tags/v1.10.0..6d9754182e5f65540957ade962cd3689eacd3599 | grep "h-vetinari"
    34  @h-vetinari
```

Reply:
Maybe that's something you can discuss with @tupui re: the machinery that calculates this in `tools/authors.py`. I'm not sure the commit counts are that useful anyway, but I probably won't have the bandwidth to debug at this time.

Reply:
It's also looking at the message to find if the name is thanked or in co-authored. So the number here should be greater or equal than the actual numbers of commit you got yourself in the history.